smoketests/tests/domains.py

```diff
@@ -3,47 +3,18 @@
 class Domains(Smoketest):
     AUTOPUBLISH = False
 
-    def test_register_domain(self):
-        """Attempts to register some valid domains and makes sure invalid domains cannot be registered"""
-
-        rand_domain = random_string()
-
-        self.new_identity()
-        self.spacetime("dns", "register-tld", rand_domain)
-
-        self.publish_module(rand_domain)
-        self.publish_module(f"{rand_domain}/test")
-        self.publish_module(f"{rand_domain}/test/test2")
-
-        with self.assertRaises(Exception):
-            self.publish_module(f"{rand_domain}//test")
-        with self.assertRaises(Exception):
-            self.publish_module(f"{rand_domain}/test/")
-        with self.assertRaises(Exception):
-            self.publish_module(f"{rand_domain}/test//test2")
-    
-    def test_reverse_dns(self):
-        """This tests the functionality of spacetime reverse dns lookups"""
-
-        rand_domain = random_string()
-        self.spacetime("dns", "register-tld", rand_domain)
-
-        self.publish_module(rand_domain)
-
-        names = self.spacetime("dns", "reverse-lookup", self.resolved_identity).splitlines()
-        self.assertIn(rand_domain, names)
-
     def test_set_name(self):
         """Tests the functionality of the set-name command"""
 
         self.publish_module()
 
         rand_name = random_string()
 
-        self.spacetime("dns", "register-tld", rand_name)
-        self.spacetime("dns", "set-name", rand_name, self.database_identity)
-        lookup_result = self.spacetime("dns", "lookup", rand_name).strip()
-        self.assertEqual(lookup_result, self.database_identity)
+        # This should throw an exception before there's a db with this name
+        with self.assertRaises(Exception):
+            self.spacetime("logs", rand_name)
+
+        self.spacetime("rename", rand_name, self.database_identity)
 
-        names = self.spacetime("dns", "reverse-lookup", self.database_identity).splitlines()
-        self.assertIn(rand_name, names)
+        # Now we're essentially just testing that it *doesn't* throw an exception
+        self.spacetime("logs", rand_name)
```

Comment:
Not to be a stickler, but would it be possible to add the following tests:

- Test that you can't rename to a database that already exists
- Test that you can't publish a database with an invalid name (e.g. that includes special characters)

I would also keep the tests that test the slash functionality. We should only remove those tests when we actually change the server behavior. It's currently testing that behavior not this CLI comamnd.

Reply:
I've added the renaming test.

What special characters should be disallowed? I tried `!@#$%^&*()` and was able to publish successfully.

In a perfect world, I would argue that the "disallowed characters" test should be more internal, i.e. testing that the endpoints will return an error when using one of those symbols. Ideally I would say the CLI tests are relatively "thin" and test that the CLI does indeed correctly trigger the endpoints we expect it to.

Reply:
I've added back separate tests for the behavior of `/`

Reply:
Well these are not solely CLI tests but just generally smoke tests, although I do agree that should be a unit test/http test as well.